demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -3295,13 +3368,17 @@ export class DemoMeetingApp
         this.eventReporter?.destroy();
       }
 
+      await this.bbprocessor?.destroy();
```

Comment:
typo? Why do need to call destroy twice :)

Reply:
I see `bb` and `br`, each with its own observer.

Reply:
Oh i cant read. Resolved :)

Reply:
These are actually 2 different variable that are bbp vs brp. I will rename the variables so that it is easier to read.